governator-servlet/src/main/java/com/netflix/governator/guice/servlet/GovernatorServletContextListener.java

```diff
@@ -82,15 +96,21 @@ public void contextDestroyed(ServletContextEvent servletContextEvent) {
     /**
      * Override this method to create (or otherwise obtain a reference to) your
      * injector.
+     * NOTE: If everything is set up right, then this method should only be called once during
+     * application startup.
      */
     protected final Injector getInjector() {
+        if (injector != null) {
+            throw new RuntimeException("We must only create an Injector once.");
```

Comment:
Actually, this should be an IllegalStateException.  Wording should be 'Injector already created'.